app/assets/src/components/ui/controls/dropdowns/BareDropdown.jsx

```diff
@@ -303,6 +316,11 @@ BareDropdown.propTypes = forbidExtraProps({
   // If search is true, but you want to customize behavior of search function, e.g. async search,
   // you should provide your own handler
   onFilterChange: PropTypes.func,
+  showNoResultsMessage: PropTypes.bool,
+  // Don't show the no results message if search options are still loading.
+  // TODO(mark): Visually indicate that search options are loading even if
+  // there are old search results to display.
+  loadingSearchOptions: PropTypes.bool,
```

Comment:
I'm a little confused by `loadingSearchOptions` (maybe the naming or the comment). If true, then it doesn't display "no results" while loading?

Reply:
Would `isLoadingSearchOptions` be better? This indicates to the dropdown that the parent component is currently waiting for a response from the server. This would actually change between true and false on the same component depending on whether we're currently waiting for a response.

Reply:
yeah that makes more sense to me